sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -322,4 +323,24 @@ export class ECertGenerationService {
       },
     });
   }
+
+  /**
+   * Get the lifetime maximums of CSLP of the student which is used in the e-Cert generation.
+   * @param assessmentId disbursement assessment id.
+   * @param entityManager used to execute the commands in the same transaction.
+   * @returns lifetime maximums of CSLP.
+   */
+  async getCSLPLifetimeMaximums(
+    assessmentId: number,
+    entityManager: EntityManager,
+  ): Promise<number> {
+    const studentAssessmentRepo =
+      entityManager.getRepository(StudentAssessment);
+    const dmnValues = await studentAssessmentRepo
+      .createQueryBuilder("studentAssessment")
+      .select("studentAssessment.workflowData ->> 'dmnValues'", "dmnValues")
```

Comment:
`dmnValues` can potentially receive more values in the future. Since the method is specifically targeting the `lifetimeMaximumCSLP`, is it possible to use the JSONB syntax to get only `lifetimeMaximumCSLP`?